api/src/main/java/org/xnio/ssl/JsseSslConduitEngine.java

```diff
@@ -317,6 +318,14 @@ private int wrap(final ByteBuffer src, boolean isCloseExpected) throws IOExcepti
                     doFlush();
                 }
             } catch (IOException ignore) {}
+            Throwable cause = e.getCause();
```

Comment:
@moulalis indentation needs fixing

Reply:
@moulalis because it is a minor thing and I need to release XNIO soon, I decided to do it myself.